rb/spec/integration/selenium/webdriver/virtual_authenticator_spec.rb

```diff
@@ -0,0 +1,395 @@
+# frozen_string_literal: true
+
+# Licensed to the Software Freedom Conservancy (SFC) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The SFC licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+require_relative 'spec_helper'
+
+module Selenium
+  module WebDriver
+    describe "VirtualAuthenticator" do
```

Comment:
Does this pass when run on all the browsers? If not, we should guard it with `only: {browser: :chrome}` or something similar (check other specs for examples).

Reply:
When I didn't use any `exclusive` parameter in my test, it _seemed_ that the test ran only on chrome. But I have made changes and used `exclusive: {browser: %i[chrome]}` now, and the test run only in chrome. For firefox and others, they are shown pending.